turbo/stages/bodydownload/body_algos.go

```diff
@@ -58,37 +56,7 @@ func (bd *BodyDownload) UpdateFromDb(db kv.Tx) (headHeight, headTime uint64, hea
 	clear(bd.requests)
 	clear(bd.peerMap)
 	bd.ClearBodyCache()
-	headHeight = bodyProgress
```

Comment:
The following code was removed since it computes values that are never used. Also this code was actually producing error messages `canonical marker not found` specified in the issue